packages/tree-widget/package.json

```diff
@@ -62,8 +62,8 @@
     "@types/react-dom": "^16.8.0",
     "@types/react-redux": "^7.0.1",
     "cpx": "^1.5.0",
-    "react": "^16.8.0",
-    "react-dom": "^16.8.0",
+    "react": "^16.13.0",
```

Comment:
Did you need to bump these versions?

Reply:
yeah I was running into issues when testing the provider in the web-viewer-test

Reply:
I think thats because you copied the entire pkg over into that repo, id revert these changes in this file since we know it works because its being used in itwin/viewer-react

Reply:
For some reason github says theres still changes to this file, might be line endings or something.

could you run:
```
git checkout origin/master packages/tree-widget/package.json
```